src/code.sh

```
@@ -8,32 +8,33 @@ dx-download-all-inputs
-fi
-
-cd arriba && make
-cd ..
-export PATH=$PATH:/home/dnanexus/arriba
+# Load arriba docker
+docker load -i $arriba_tar_path
+DOCKER_IMAGE_ID=$(docker images --format="{{.Repository}} {{.ID}}" | grep "^uhrigs/arriba" | cut -d' ' -f2)
 
 #Extract sample name from input_bam
-sample_name=$(echo $bam_prefix | cut -d '_' -f 1)
+sample_name=$(echo $bam_prefix | cut -d '.' -f 1)
```

Shellcheck (0.10.0)

[warning] bam_prefix is referenced but not assigned.

(SC2154)